lib/graphql_authorize/configuration.rb

```diff
@@ -7,6 +7,9 @@ class Configuration
     def initialize
       @auth_adapter = nil
       @auth_adapter_source = nil
+
+      I18n.load_path << Dir[File.expand_path("config/locales") + "/*.yml"]
```

Comment:
I've found one more thing - if no one calls configuration then this code won't be executed. Thoughts on moving it somewhere else, like https://github.com/anjlab/graphql_authorize/blob/master/lib/graphql_authorize.rb ?

Reply:
Very good point/spot. Yep; I'll move it for exactly that reason.